### PR TITLE
Add descriptions to schema fields.

### PR DESCRIPTION
**Pull request by CalvinScott**

### CODE REVIEW

**Comment by CalvinScott** [comment]
@CMClay @RongxinZhang Ready for review. Instead of reading them all, maybe just check to see if my description of all the IDs is somewhat accurate. The DD definitions of the IDs are even more vague.

File: lib/listing-desc.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  streetName: 'The street name for the listing property.',
+  streetNumber: 'The street number for the listing property.',
+  city: 'The city which the listing property resides in.',
+  state: 'The state which the listing property resides in.',
+
+  county: 'The county with which the listing property resides in.',
+  country: 'The country with which the listing property resides in.',
+  zipCode: 'The postal code for the listing property.',
+  subdivision: 'The name of the subdivision if the listing property has one.',
+  _geo: 'The type and coordinates of the geospacial location of the listing property.',
```
Comment (CMClay) [inline lib/listing-desc.js]: Fields prefixed with `_` are used for internal purposes. Having a description is fine, but we need to make sure we remove them from the swagger.json.

**Comment by CMClay** [comment]
Nice work @CalvinScott! Can you add a test to make sure the relevant fields have descriptions?

**Comment by CalvinScott** [comment]
@CMClay Added a simple test

**Comment by CMClay** [comment]
:+1:

**Comment by RongxinZhang** [comment]
:+1: looks good

**Comment by CalvinScott** [comment]
Waiting on https://github.com/Retsly/retsly-schemas/pull/18 before merge

**Comment by CalvinScott** [comment]
@lvivier Take a look!

File: lib/listing-desc.js (hunk 1 above)
Comment (slajax) [inline lib/listing-desc.js]: I agree with @CMClay, would it not be appropriate to just omit them from retsly-schema?

**Comment by lvivier** [comment]
:+1:

**Comment by slajax** [comment]
I am signing off on merging this to be released and installed to www / api.

**Pull request merged**